src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure/LRO/Base/AzureLRO.cs

```diff
@@ -223,7 +223,10 @@ protected virtual void UpdatePollingState()
                 }
                 else
                 {
-                    throw new CloudException("The response from long running operation does not have a valid status code.");
+                    throw new CloudException("The response from long running operation does not have a valid status code.")
+                    {
+                        HttpStatusCode = (int)CurrentPollingState.CurrentStatusCode
```

Comment:
Any reason for using PollingState.StatusCode.
This can be the status code from the previous polling (which would be incorrect)
The message for the exception is that, the LRO response had no status code.